fuzz_write/src/main.rs

```diff
@@ -49,11 +49,11 @@ pub struct FileOperation<'k> {
     // 'abort' flag is separate, to prevent trying to copy an aborted file
 }
 
-impl<'k> FileOperation<'k> {
+impl FileOperation<'_> {
     fn get_path(&self) -> Option<PathBuf> {
         match &self.basic {
             BasicFileOperation::SetArchiveComment(_) => None,
-            BasicFileOperation::WriteDirectory(_) => Some(self.path.join("/")),
+            BasicFileOperation::WriteDirectory(_) => Some(self.path.join("")),
```

Comment:
I am not sure whether this is intened. I guess in this line we want to return a path with trailing `/`, so I replaced the params with `""`.

see: https://rust-lang.github.io/rust-clippy/master/index.html#join_absolute_paths

Reply:
This probably isn't the correct behavior either. What's the correct way to append a `/` to a path? That's how we detect that it is, in fact, intended to be a directory and not a normal file.

Reply:
> The documentation indicates, if I'm reading it correctly, that `self.path.join("")` doesn't do that and is instead a no-op.

I think `self.path.join("")` appends a `/` at the end of path. You can verify that with following code:

```rs
use std::path::PathBuf;

fn main() {
    let original_path = PathBuf::from("/home/user");
    let joined_path = original_path.join("");

    println!("Original path: {:?}", original_path);
    println!("Joined path: {:?}", joined_path);
    assert_eq!(joined_path, PathBuf::from("/home/user/"));
}
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=37109e0971e2e92e6ef255d46d3f8eeb